addons/logistics_wirecutter/script_component.hpp

```diff
@@ -88,7 +88,7 @@
 #define CUT_TIME_ENGINEER 7.5
 
 #define HAS_WIRECUTTER(unit) (\
-    "ACE_wirecutter" in (unit call EFUNC(common,uniqueItems)) \
+    count ((unit call EFUNC(common,uniqueItems)) select {_x isKindOf "ACE_wirecutter"}) > 0 \
```

Comment:
```suggestion
    count ((unit call EFUNC(common,uniqueItems)) select {_x isKindOf ["ACE_wirecutter", configFile >> "CfgWeapons"]}) > 0 \
```

`isKindOf` needs [alternate syntax](https://community.bistudio.com/wiki/isKindOf) for CfgWeapons.

Reply:
My bad, forgot about that part